spec/bolt/transport/ssh_spec.rb

```diff
@@ -515,9 +516,24 @@ def result_value(stdout = nil, stderr = nil, exit_code = 0)
       it "returns a failed result", ssh: true do
         expect {
           ssh.run_command(target, 'whoami')
-             .to raise_error(Bolt::Node::EscalateError,
-                             "Sudo password for user #{user} was not provided for #{hostname}:#{port}")
-        }
+        }.to raise_error(Bolt::Node::EscalateError,
+                         "Sudo password for user #{user} was not provided for #{hostname}:#{port}")
+      end
+    end
+
+    context "as bash user with no password" do
+      let(:config) {
+        mk_config('host-key-check' => false, 'run-as' => 'root', user: bash_user, password: bash_password)
+      }
+
+      it "returns a failed result when a temporary directory is created", ssh: true do
```

Comment:
This test doesn't seem to fail when I revert the corresponding code changes.

Reply:
Or was this just intended as added coverage for an existing case, rather than a test of the new behavior?

Reply:
I spoke too soon. It does fail (well, hangs) if I revert *both* the `chown` change and the sudo change. I had previously only reverted one.